src/Core/Resources/Resources.cpp

```diff
@@ -3,59 +3,48 @@
 #include <Core/Utils/Log.hpp>
 #include <algorithm>
 #include <cpplocate/cpplocate.h>
+#include <filesystem>
```

Comment:
Well, I'm not sure it's what the doc says.
The variable is on the cmake side.

So we could either check that fs.Final is found, or do something like
add_compile_definition(FILESYSYTEM_EXPERIMENTAL_OR_FINAL  ...)

and 
```suggestion
#ifdef FILESYSTEM_FINAL
#include <filesystem>
#else
#include <experimental/filesystem>
#endif
```
But I'm not sure it's better.

By the way CI pass on the 3 arch with gcc-8 on linux.

Reply:
Your right, but then I prefer to add the following compile definition in the `CMakeFiles.txt`
`add_compile_definition(CXX_FILESYSTEM_HEADER=${CXX_FILESYSTEM_HEADER})` if the module is found and do not make the source heavier than needed by using `#include < CXX_FILESYSTEM_HEADER >` instead of `ifdef ... else ... endif`

Reply:
what about https://stackoverflow.com/questions/53365538/how-to-determine-whether-to-use-filesystem-or-experimental-filesystem

Reply:
Discussing in real ?